src/app/components/PopupDuration/index.tsx

```diff
@@ -0,0 +1,89 @@
+import { LoadingButton } from "@mui/lab";
```

Comment:
Why do you need to create this component? If you want to create a custom popup it has already been defined in **_ModalCustomization_** component